src/pages/trainer/training/edit/components/toolbar/icons/italicIcon.tsx

```diff
@@ -0,0 +1,7 @@
+import * as React from 'react';
+
+export const ItalicIcon = () => {
+  return (
+    <span className="glyphicon glyphicon-italic"/>
+  );
+};
```

Comment:
Most of this icons are very similar not sure if we could just use a baseIcon component plus an enum to define which icon to use, similar approach to dashboard, what do you think?